apps/versions/tests.py

```diff
@@ -1237,6 +1238,24 @@ def test_file_not_multi_package(self):
         files = version.all_files
         assert not files[0].is_multi_package
 
+    def test_track_upload_time(self):
+        # Set created time back (just for sanity) otherwise the delta
+        # would be in the microsecond range.
+        self.upload.update(created=datetime.now() - timedelta(days=1))
+
+        with mock.patch('versions.models.statsd.timing') as mock_timing:
+            Version.from_upload(self.upload, self.addon, [self.platform])
+
+            upload_start = calendar.timegm(
+                self.upload.created.utctimetuple())
+            now = calendar.timegm(datetime.now().utctimetuple())
+            rough_delta = (now - upload_start) * 1000
+            actual_delta = mock_timing.call_args[0][1]
+
+            fuzz = 2000  # 2 seconds
+            assert (actual_delta >= (rough_delta - fuzz) and
+                    actual_delta <= (rough_delta + fuzz))
```

Comment:
I have no idea what that does but if you promise it does the same thing I can add it